enrolments/tests/test_validators.py

```diff
@@ -52,6 +53,73 @@ def test_validate_classes_in_session(self):
             session_other,
         )
 
+    def test_validate_enrolment_in_session(self):
+        session1 = Session.objects.create(season=Session.SPRING, year="2020")
+        session2 = Session.objects.create(season=Session.FALL, year="2019")
```

Comment:
```suggestion
        session2 = self.session
```
since there's already a session that exists